apps/web/next.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = transpileModules(
     },
     webpack: (config) => {
       config.experiments = {
+        ...config.experiments,
```

Comment:
Found that overwriting the experiments object was blowing away some of Next's optimizations under the hood and causing errors.